crowbar_framework/app/models/node_object.rb

```diff
@@ -1207,7 +1207,7 @@ def run_ssh_cmd(cmd)
     args = ["sudo", "-i", "-u", "root", "--", "timeout", "-k", "5s", "15s",
             "ssh", "-o", "ConnectTimeout=10",
             "root@#{@node.name}",
-            cmd.to_s
+            "\"#{cmd}\""
```

Comment:
I just think whoever wants to pass the quotes to the command must do more testing if the command actually works. It should be possible to use `\\\"` for example, or single quotes, or combination.